packages/manager/apps/telecom/src/app/telecom/pack/move/eligibility/lineNumber/move-eligibility-lineNumber.controller.js

```diff
@@ -30,6 +30,19 @@ export default class {
    * @param { String } status status of the line: 'active' | 'inactive'
    */
   getLineCopper(lineNumber, status) {
+    if (this.isReseller) {
+      // If current offer is a reseller offer,
+      // launch eligibility test line for partners (reseller)
+      return this.OvhApiConnectivityEligibility.v6()
+        .testLinePartners(this.$scope, {
+          lineNumber,
+          status,
+        })
+        .catch((error) => {
+          this.loading = false;
+          this.TucToast.error(error);
+        });
```

Comment:
Seems to be the same code as below, the only things that change is the `testLinePartners`/`testLine`.

Reply:
Yes the call change if is reseller or not

Reply:
Maybe this could be changed, to only set `testLinePartners / testLine` as the promise to be used.